src/components/common/AgreementType.vue

```diff
@@ -102,15 +101,9 @@ export default class AgreementType extends Mixins(EnumMixin) {
       .find(x => x.code === this.getIncorporationAgreementStep.agreementType)?.description
   }
 
-  // Lifecycle methods
+  /** Called when component is created. */
   created (): void {
-    // temporarily ignore data changes
-    this.setIgnoreChanges(true)
     this.agreementType = this.getIncorporationAgreementStep.agreementType
-    // watch data changes once page has loaded (in next tick)
-    Vue.nextTick(() => {
-      this.setIgnoreChanges(false)
-    })
```

Comment:
The original reason for this code was to ignore changes to the store when this component started up (so that we would only tell the user they have unsaved changes if they made changes -- not changes due to component init).

However, there are no changes to the store in created() here, so that code is not needed.